apis/v1alpha1/gatewayclass_types.go

```diff
@@ -103,7 +114,7 @@ type GatewayClassStatus struct {
 	// +listType=map
 	// +listMapKey=type
 	// +kubebuilder:validation:MaxItems=8
-	// +kubebuilder:default={{type: "InvalidParameters", status: "Unknown", message: "Waiting for controller", reason: "Waiting", lastTransitionTime: "1970-01-01T00:00:00Z"}}
+	// +kubebuilder:default={{type: "Accepted", status: "Unknown", message: "Waiting for controller", reason: "Waiting", lastTransitionTime: "1970-01-01T00:00:00Z"}}
```

Comment:
I think it may be clearer to start off with a false state.
```suggestion
	// +kubebuilder:default={{type: "Accepted", status: "false", message: "Waiting for controller", reason: "Waiting", lastTransitionTime: "1970-01-01T00:00:00Z"}}
```

Reply:
I thought that the defaulting behavior for conditions is supposed to be `Unknown` until the relevant controller has seen it? In Contour's case, it's possible for us to set `False`, so the distinction between `Unknown` and `False` is relevant.

Reply:
I think "Unknown" made a lot more sense when the condition was "InvalidParameters". Now that we have switched to "Admitted" I think it's reasonable to start in a state where "Admitted" is false with a "Waiting" reason. Controllers should switch it to true or change the reason it is false if/when they process the class. I think `admitted == false` means that there is no controller that will support provisioning gateways of this class. That seems like a reasonable default state until a controller picks this up and changes it.

Reply:
Yeah, that makes sense. I'll change this up.